tools/create-app.bash

```diff
@@ -5,10 +5,23 @@
 
 # Params: $1: output directory
 
-# Sign app for release
+### Assets ###
+
+# Place logo in the assets file
+full_path=$(realpath $0)
+dir_path=$(dirname $full_path)
+mv ./logo-hippie.png $dir_path/../app/assets/images/
+
+# Generating splash screen
+tns resources generate splashes <./splash-image.png> --background <rgb-color>
+
+# Generating app icon
+tns resources generate icons <./icon-image.png>
```

Comment:
The icon image should be searched as `icon.png` from a given images directory, as an argument.